.github/test-spec.yml

```diff
@@ -517,3 +517,27 @@
       - "subsys/nrf_security/**/*"
       - "!subsys/nrf_security/doc/**/*"
       - "!subsys/nrf_security/*.rst"
+
+"CI-suit-dfu-test":
+  - "include/bl*"
+  - "include/fprotect.h"
+  - "include/fw_info.*"
+  - "include/mgmt/*"
+  - "include/dfu/**/*"
+  - "include/net/**/*"
+  - "cmake/*"
+  - "modules/mcuboot/**/*"
+  - "modules/trusted-firmware-m/*"
+  - "subsys/bootloader/**/*"
+  - "subsys/nonsecure/*"
+  - "subsys/partition_manager/*"
+  - "subsys/dfu/*"
+  - "subsys/fw_info/**/*"
+  - "subsys/mgmt/**/*"
+  - "subsys/pcd/*"
+  - "subsys/net/lib/*fota*/**/*"
+  - "subsys/net/lib/download_client/**/*"
+  - "subsys/suit/**/*"
+
+"CI-suit-dfu-test-extended":
+  - "subsys/suit/**/*"
```

Comment:
Missing directories which should trigger extended scope:
  - "subsys/mgmt/suitfu/**/*"
  - "samples/suit/**/*"